### DISCUSSION TITLE
[FEATURE]应用商店umami的docker地址更新

### DISCUSSION THREAD

**Discussion by xiaobaozi233** [1Panel-dev/1Panel#5091] [Ideas]
### 1Panel 版本

V1.10.6

### 请描述您的需求或者改进建议

umami的地址已经变成了
`docker.umami.is/umami-software/umami`
应用商店里的是
`ghcr.io/umami-software/umami`
ghcr这阿里云实在没速度

**Comment by wanghe-fit2cloud** [comment]
在不添加任何镜像加速的情况下，新的镜像仓库下载镜像失败，如图：
![image](https://github.com/1Panel-dev/1Panel/assets/46734380/c6817a0f-e31e-4bca-90f2-9f235f7310cc)

**Reply by xiaobaozi233** [reply]
仔细看了一下ghcr.io是谷歌的，docker.umami.is是官方用AWS部署的。
我是使用的阿里云镜像直接部署的1Panel。
这两个的都能访问，但是速度半斤八两。
之前ghcr.io有过30分钟以上还没拖下来的情况。
